Add a catalog of implementation recipes (#6408)
PrefectHQ/prefect: docs/recipes/recipes.md

---
description: Check out the implementation recipes collected in the Prefect Recipes library.
tags:
---

# Prefect Recipes

> [Prefect Recipes](https://github.com/PrefectHQ/prefect-recipes) provide the ingredients for common and extensible patterns for setting up Prefect in your execution environment with readymade resources like dockerfiles, Terraform files, GitHub actions, and more.

Reviewer: ```suggestion
[Prefect Recipes](https://github.com/PrefectHQ/prefect-recipes) are common, extensible patterns for setting up Prefect in your execution environment with readymade resources like dockerfiles, Terraform files, GitHub actions, and more.
```

---
description: Check out the implementation recipes collected in the Prefect Recipes library.
tags:
---

# Prefect Recipes

[Prefect Recipes](https://github.com/PrefectHQ/prefect-recipes) are common, extensible patterns for setting up Prefect in your execution environment with readymade resources like dockerfiles, Terraform files, GitHub actions, and more.

The following are specific Prefect Recipes for Prefect 2.0. You can find the full repository of Recipes at [https://github.com/PrefectHQ/prefect-recipes](https://github.com/PrefectHQ/prefect-recipes).

### Contributing

We're always looking for new recipe contributions! See the [Prefect Recipes](https://github.com/PrefectHQ/prefect-recipes#contributing--swag-) repository for details on how you can add your Prefect 2.0 recipe, share flow best practices with fellow Prefect users, and earn some swag.

> ## Recipe catalog

Reviewer 2: ```suggestion
## Recipe Catalog
```

Author: Sentence case is our standard usage for H2 and lower headings. Prefect Collections is a branded usage. I'm not sure if we want to lean into Recipe Catalog as a branded usage, but will keep this in mind as we develop the concept.
